README.md

```diff
@@ -1 +1,257 @@
-# to-the-moon-be
\ No newline at end of file
+# To the Moon and Backend 
+
+This is the final project at Turing School of Software and Design in Denver, CO. It is the backend portion of a service-oriented architecture. Working directly with a front end team, we were provided a detailed wireframe and a list of expected responses. We built an API that consumes multiple external APIs to gather and format that data required. 
+Stack: Python3, Flask, Postgres, Heroku 
+
+# Endpoints 
+
+### Celestial Bodies Index Endpoint
```

Comment:
We could probably include fewer celestial bodies and just use '. . . ' to signify that there would be more to avoid clutter.

Reply:
Good call. Ill make that update and create a new PR.